PlayCover/View/PlayAppView.swift

```diff
@@ -64,7 +64,12 @@ struct PlayAppView: View {
                     Text("Show in Finder")
                     Image(systemName: "folder")
                 })
-
+                Button(action: {
```

Comment:
It would be more appropriate to use a checkbox `Toggle` here instead of a button.

Reply:
> It would be more appropriate to use a checkbox `Toggle` here instead of a button.

Simple to explain. First of all I don't do that because it would involve creating a new state and due to the way the app menu is created, we'll have to do all kinds of funny business just to change it to a toggle

Second is that this happens if you use a Toggle.

<img width="280" alt="image" src="https://user-images.githubusercontent.com/23693150/181955567-ea9195cf-1fc5-4edb-b37b-fa8f7ba1df4d.png">

<img width="235" alt="image" src="https://user-images.githubusercontent.com/23693150/181955824-27dc0439-9931-49c5-a006-905b930e3ac7.png">

Not ideal.

Reply:
Oh, this is in the right click menu. It would be better to put it in the App Settings View instead

Reply:
Moved to App Settings. Someone need to clean that thing up though. We are already running at the 10 views limit